custom/icds_reports/tests/test_inactive_dashboard_users.py

```diff
@@ -41,10 +41,11 @@ def test_get_inactive_users_data_added(self):
                 csv_reader = csv.reader(items_file)
                 data = list(csv_reader)
                 sorted(data, key=lambda x: x[0])
-                self.assertEqual([['Username', 'Location', 'State'],
```

Comment:
What is the error that was happening that lead to this change? @nickpell

Reply:
https://api.travis-ci.org/v3/job/530343209/log.txt